### ISSUE TITLE
Ability to increase margin between line number and fold indicators in gutter

### ISSUE THREAD

**Issue by smileatom** [bobbylight/RSyntaxTextArea#327]
It just looks too cramped to me... Is there any way to do this?

**Comment by bobbylight** [comment]
Not at the moment, though I suppose you could fake it out by creating your own fold indicator icons with some amount of padding on the left-hand side:

```java
RTextScrollPane sp = ...;
sp.getGutter().setFoldIcons(collapsedIcon, expandedIcon);
```
This would probably be pretty simple so I might tackle it in the next release, but pull requests are welcome as well!

**Comment by bobbylight** [comment]
This features is available in the most recent `3.0.9-SNAPSHOT` in Sonatype.  Set this value as follows (the default is `0`):

```java
RTextScrollPane sp = ...;
sp.getGutter().setSpacingBetweenLineNumbersAndFoldIndicator(5);
```
